mon-entreprise/source/pages/Stats/GlobalStats.tsx

```diff
@@ -105,7 +105,7 @@ export default function GlobalStats({ stats }: { stats: StatsStruct }) {
 	const last30dConv = Math.round((100 * last30dCommenceNum) / last30dVisitsNum)
 
 	const last30dSatisfactions = stats.satisfaction
-		.filter(({ date }) => lastCompare(day30before, date))
+		.filter(({ date }) => date.startsWith(new Date().toISOString().slice(0, 7)))
```

Comment:
Attention le code de `GlobalStats` fait encore explicitement l'hypothèse qu'on récupère les notes de satisfaction des 30 derniers jours, dans le nom de ses variables.

Aucun soucis pour que je fasse un petit commit de clean-up étant donné que c'est mon code, mais je n'ai pas été mentionné dans cette PR ni demandé comme reviewer donc j'aurais pu passer à côté.

En fait, je vais regarder très rapidement, mais il me semble qu'il y a un soucis initial dans le code de `prepare.js`: les objets de `stats.satisfaction` ont des attributs `date`, ce qui est sémantiquement imprécis. On devrait parler de `month`.

Reply:
Oups, c'est noté, je te mentionnerai la prochaine fois que je modifie du code que tu as écris.

Complètement d'accord pour l'imprécision, il s'agit bien de mois et pas de date.